### PR TITLE
Updates required to make final version of DR7 products

### PR DESCRIPTION
**Pull request by jotaylor**
- Update README with more accurate install & product instructions. NOTE: I am not finished with README updates, just wanted to commit what I already have.
- Improve target name matching when it is needed for HLSP filenames and headers.
- Change stisblazefix wrapper to accept an output _file_ rather than a directory. This allows the user to specify and extension and write an x1d rather x1f if desired. The x1fs were causing some confusion for the HLSP creation code.
- Consistent usage of the text `WARNING` in all caps. This helps me with checking the output from HLSP creation code when grepping.
- Change `EPOCH` keyword to `G_EPOCH`
- Adjust starting wavelength for abutting STIS/E140M to avoid very noisy region.
- Remove obsolete calls to `get_coords` in ullyses coadd
- Update package requirements, these were required for a fresh install.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -34,6 +35,7 @@ dependencies = [
     "matplotlib",
     "pyyaml",
     "plotly",
+    "scipy>=1.11.4",
     "hasp>=0.9.5"
```
Comment (stscirij) [inline pyproject.toml:39]: hasp>=0.9.6

**Review by stscirij — CHANGES_REQUESTED** [review]
This all looks good, I merged the latest hasp code and tagged as version 0.9.6, so that is the version that should go into the pyproject.toml.   You should definitely test that it still gives exactly the answers you expect

**Comment by jotaylor** [comment]
I tested two example targets, AV-456 and CVSO-58. Products for both targets were identical using new HASP version vs. DR7 HLSPs created last week.

**Review by stscirij — APPROVED** [review]
LGTM

**Pull request merged**